### DISCUSSION TITLE
Low battery on VE.Bus System after switch to LiefPo4 system

### DISCUSSION THREAD

**Discussion by hansgeorgk** [Louisvdw/dbus-serialbattery#144] [Q&A] [answered]
Hi,

since I changed to LifePo4 System with Daly BMS and EVE cells I am seeing this error "Low battery on VE.Bus System" quite often. System is is switching off and is recovering after a few seconds.
Seems like a short peak, when starting eg the circular saw and having other devices running at the same time, is causing this. 
I have set:
MAX_BATTERY_DISCHARGE_CURRENT = 200.0 in utils.py, which is the same value I have set in Daly BMS.
Is this the limiting factor?
If yes, can I increase this value to a higher number in utils.py?

cheers
HG

**Comment by Louisvdw** [comment, marked as answer]
Many device have a power spike when you start then. This is normally high with anything that have a motor that start and can be as high as 7x the normal power rating of that motor. It is for this reason also that Victron Inverters have a high capacity peak, but can only give this peak power for a few seconds. 
Your battery needs to be able to handle this peak, or it will protect itself and disconnect.
So for instance the MP II 5000 can give 4000W continously at 25°C, but it has a peak capacity of 9000W for a few seconds. 

For the battery to be able to supply that, your BMS need to be able to handle 180A (for 48V battery), or 750A (for 12V battery). Anything more will use utility power if grid tied or disconnect if off grid. For your battery to handle that your BMS must be rated for that and your cells as well. 

The discharge current in utils.py need to match what your BMS can do. If it is more then the system will try to draw that power and trip the battery.
